test/assert.fail.js

```diff
@@ -1,3 +1,23 @@
 const assert = require('../');
 
-assert.fail('actual', 'expected', 'Should never pass', 'assert.fail');
+const actual = 'actual';
+const expected = 'expected';
+const operator = '<=>';
+
+assert.fail();
+assert.fail(actual, expected, 'with custom message');
+assert.fail(actual, expected, 'with message and operator', operator);
+assert.fail(actual, expected, undefined, operator);
+
+(function stackStart() {
+  assert.fail(actual, expected, 'with everything', operator, stackStart);
+})();
+
+(function stackStart() {
+  assert.fail(actual, expected, undefined, operator, stackStart);
+})();
+
+if (process.version >= 'v8.0.0') {
```

Comment:
Should I add a https://github.com/npm/node-semver dependency?

Reply:
Either that, or use a regex like `if (!/^v[0-7]\./.test(process.version))`.  This will start failing at node v10.

Reply:
Which do you prefer?

Reply:
Adding the first dependency feels almost wrong? Maybe I'm just too new to npm culture in general.

Reply:
RegExp approach is fine.  That's what I'd probably do, for such a simple test.

Reply:
Yep. I used your regexp verbatim.